content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md

```diff
@@ -0,0 +1,175 @@
+---
+Title: Redis Enterprise for Kubernetes release notes 6.4.2-tbd (June 2023)
+linktitle: 6.4.2-tbd (June 2023)
+description: The Redis Enterprise K8s 6.4.2-tbd release supports Redis Enterprise Software 6.4.2 and contains new features and feature improvements.
+weight: 55
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [ 
+    /kubernetes/release-notes/6-4-2-releases/k8s-6-4-2-tbd/,
+      ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.4.2-tbd release supports Redis Enterprise Software 6.4.2 and contains new features and feature improvements.
+
+The key features, bug fixes, and known limitations are described below.s
+
+## Images
+
+* **Redis Enterprise**: `redislabs/redis:6.4.2-81`
+* **Operator**: `redislabs/operator:6.4.2-tbd`
+* **Services Rigger**: `redislabs/k8s-controller:6.4.2-tbd.`
+
+### OpenShift images
+
+* **Redis Enterprise**: `registry.connect.redhat.com/redislabs/redis-enterprise:6.4.2-81.rhel8-openshift`
+    (or `redislabs/redis-enterprise:6.4.2-81.rhel7-openshift` if upgrading from RHEL 7)
+* **Operator**: `registry.connect.redhat.com/redislabs/redis-enterprise-operator:6.4.2-tbd`s
+* **Services Rigger**: `registry.connect.redhat.com/redislabs/services-manager:6.4.2-tbd`
+
+### OpenShift OLM bundles
+
+* Redis Enterprise operator bundle version: `v6.4.2-tbd`
+
+## New features
+
+* General availability of the Active-Active database controller for declarative configuraiton support
+
+## Feature improvements
+
+* Active-Active controller improvements include adding support for the following:
+  * Configuration via OLM
+  * Certificates in `globalConfiguration`
+  * Backups in `globalConfiguration`
+  * Modules (preview)
+  * Hashicorp Vault secrets
+  * indication of sync failures to REAADB
+* Added support for separate `podAnnotations` for Redis Enterprise pods to allow setting `appArmor` exceptions
+* Timezone can be passed as environment variable into Redis Enterprise pods
+* REDB namespaces can be labelled without the operator going into `CrashLoopBackoff`
+* Added support for K8s 1.27
+* Added support for K8s 1.25 for EKS
+* Added support for K8s 1.26 for AKS and GKE
+* Improved log collector execution log, filtering out empty files, capturing volume attachments, option to disable RS capture debug info package
+* Added `extraEnvVars` field in REC to support environment variables for RS pods
+
+## Fixed bugs
+
+* REC in invalid state can't be deleted (RED-78124)
+* OpenShift route not deleted after REC is deleted (RED-94121)
+* REAADB secret status not being updated in source cluster (RED-96296)
+* Invalid REAADB that is not rejected by admission might be get deleted after apply (RED-96300)
```

Comment:
This sentence was a bit confusing. Aside from the verb fix, should "admission" be "admission controller" and should "after apply" be either "after application" or "after applying" something specific? 
```suggestion
* Invalid REAADB that is not rejected by admission might be deleted after apply (RED-96300)
```